src/main/java/com/atlassian/db/replica/internal/ConnectionParameters.java

```diff
@@ -72,10 +76,19 @@ public void setHoldability(Integer holdability) {
         this.holdability = holdability;
     }
 
+    public boolean isReadOnly() {
+        return readOnly != null && readOnly;
```

Comment:
So `false` by default, but we cannot inline that in the field initializer, because we need the `null` in `initialize`